src/main/java/hyperledger/besu/java/rest/client/controller/EthClientController.java

```diff
@@ -0,0 +1,47 @@
+package hyperledger.besu.java.rest.client.controller;
+
+import hyperledger.besu.java.rest.client.model.ClientResponseModel;
+import hyperledger.besu.java.rest.client.service.TransactionService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+public class EthClientController {
+
+  @Autowired TransactionService transactionService;
+
```

Comment:
We should have a decode API  for transaction, which was discussed earlier.

Reply:
Sure, in upcoming PRs.